public/index.pug

```diff
@@ -29,22 +29,36 @@ html(lang="en-US")
     header(class="header")
       ul(class="header__list")
         li(class="header__list-item")
-          a(href="https://github.com/simple-icons/simple-icons") GitHub
+          a(class="header__list-icon" title="repository" href="https://github.com/simple-icons/simple-icons" rel="noopener") 
```

Comment:
Could you remove the `rel="noopener"` here and in the other header links. If there's no `target` specified than it's not needed, see also #83.

Reply:
removed

Reply:
Looks like you removed it everywhere, not only in these header links :confused: 

Also, minor (and unfortunate that Prettier doesn't pick this up), but now there's a space between the last quote and the closing parenthesis of the `.header__list-icon` elements.

Reply:
done correcting that.